### PR TITLE
Show node health status in crm_mon output

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review this one too? Thanks

File: lib/pengine/pe_output.c [hunk 1]
```diff
@@ -1493,20 +1492,64 @@ node_html(pcmk__output_t *out, va_list args) {
...
+                node_mode = "OFFLINE (maintenance)";
+            }
+
+        } else if (node->details->online) {
+            node_mode = "online";
+
+        } else {
+            node_mode = "OFFLINE";
+        }
+
```
Comment (clumens) [inline lib/pengine/pe_output.c:1574]: This feels like its own static function to me.

File: xml/api/nodes-2.18.rng [hunk 2]
```diff
@@ -16,6 +16,15 @@
...
             <attribute name="unclean"> <data type="boolean" /> </attribute>
+            <optional>
+                <attribute name="health">
+                    <choice>
+                        <value>red</value>
+                        <value>yellow</value>
+                        <value>green</value>
+                    </choice>
+                </attribute>
+            </optional>
```
Comment (clumens) [inline xml/api/nodes-2.18.rng:27]: You're always adding health in node_xml, so this doesn't have to be optional - unless you want this schema to validate older stuff too.

Comment (kgaillot) [reply xml/api/nodes-2.18.rng:27]: Yep, schemas should always be backward compatible (unless we bump the schema major version, which should be avoided)

**Comment by clumens** [comment]
This looks good to me now.

**Pull request merged**